lte/gateway/python/magma/pipelined/app/classifier.py

```diff
@@ -176,9 +195,31 @@ def _install_internal_conntrack_flow(self):
                                              reset_default_register=False,
                                              resubmit_table=self.config.internal_conntrack_fwd_tbl)
 
+    def _send_message_interface(self, ue_ip_add: str, local_f_teid: int):
+        """
+        Send the flow statictics request to ryu controller for
+        fetching the flow information from ovs table 14 for
```

Comment:
update docstring as we're not checking tbl 14 anymore

Reply:
Corrected the docstring.